workspaces/adr/plugins/adr/src/components/AdrReader/AdrReader.tsx

```diff
@@ -97,11 +94,13 @@ export const AdrReader = (props: {
           !error &&
           !backendUrlError &&
           value?.data && (
-            <MarkdownContent
+            <AdrMarkdownContent
               content={adrContent}
-              linkTarget="_blank"
-              transformImageUri={href => {
-                return `${backendUrl}/image?url=${href}`;
+              urlTransform={(url: string) => {
```

Comment:
any reason not to rely on the builtin `transformImageUri` prop instead? I believe that would remove the need to explicitly match specific file extensions for images